effect/shared/src/main/scala/scalaz/effect/IO.scala

```diff
@@ -502,28 +603,44 @@ object IO extends IOInstances {
    * will be undefined and most likely involve the physical explosion of your
    * computer in a heap of rubble.
    */
-  final def suspend[A](io: => IO[A]): IO[A] = Suspend(() => io)
+  final def suspend[E, A](io: => IO[E, A]): IO[E, A] = Suspend(() => io)
 
   /**
    * Imports a synchronous effect into a pure `IO` value. If the thunk returns
    * a null value, this will throw a `NullPointerException` inside `IO`.
    *
    * {{{
-   * def putStrLn(line: String): IO[Unit] = IO.sync(println(line))
+   * def putStrLn(line: String): IO[Throwable, Unit] = IO.partialSync(println(line))
```

Comment:
This scaladoc is for `sync`, yet it mentions `partialSync`.

Reply:
Fixed, thanks!